### PR TITLE
MTM-57484-Unify-usage-of-must-vs-have-to

### PR DESCRIPTION
**Pull request by dianaolives**
Replace usage of “have/has to” and “need to” with “must” wherever it does not change the meaning.

### CODE REVIEW

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 1]
```diff
@@ -276,7 +276,7 @@ You must now use the
 [softwareag/apama-cumulocity-builder](https://hub.docker.com/r/softwareag/apama-cumulocity-builder) image as a builder image.
 To do this with the default project Dockerfile created by {{< sag-designer >}} in 10.15.0 and previous versions,
 you must either change the `FROM` lines in the Dockerfile appropriately
-(you only need to do this once) or build using the following flags (you have to do this every time):
+(you only must do this once) or build using the following flags (you must do this every time):
```
Comment (kailol-sag) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:279]: "(you only must do this once)": The "must" sounds odd here. 
I checked your change in DeepL, but it always suggests the "need to" variant. 
So can we please keep the original "(you only need to do this once)"?

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 2]
```diff
@@ -665,7 +665,7 @@ Sending an event is done by constructing the event, either with `new <type>` fol
 
 #### Listening to events {#listening-to-events}
 
-You can trigger your EPL by listening to events on channels. You can subscribe to channels with the `monitor.subscribe("string name")` method. This can be done in the startup of your monitor, or if you only need to receive events some of the time, called as needed, followed by `monitor.unsubscribe("string name")`.
+You can trigger your EPL by listening to events on channels. You can subscribe to channels with the `monitor.subscribe("string name")` method. This can be done in the startup of your monitor, or if you only must receive events some of the time, called as needed, followed by `monitor.unsubscribe("string name")`.
```
Comment (kailol-sag) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:668]: I would not change this to "must". But if you really want to get rid of the "need to", then it would be better to say "if you only want to receive ...".

File: content/concepts/introduction-bundle/api.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ weight: 70
...
 
-In contrast to many other IoT platforms, {{< product-c8y-iot >}} uses the same APIs and the same interface technology for all use cases. As a consequence, you have a wider range of choices in putting intelligence into your IoT devices, depending on how powerful they are. Moreover, you only need to use one set of APIs and one technology to build a complete solution from device to application on your own.
+In contrast to many other IoT platforms, {{< product-c8y-iot >}} uses the same APIs and the same interface technology for all use cases. As a consequence, you have a wider range of choices in putting intelligence into your IoT devices, depending on how powerful they are. Moreover, you only must use one set of APIs and one technology to build a complete solution from device to application on your own.
```
Comment (BeateRixen) [inline content/concepts/introduction-bundle/api.md:8]: Below you use "must only", here you use "only must". Please make it consistent in the one or the other way :-)

File: content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md [hunk 4]
```diff
@@ -50,14 +50,14 @@ Click **Next** to proceed with the next configuration step. Click **Cancel** to
...
-* It needs to start with an alphanumeric character (letters and numbers).
+* It must start with an alphanumeric character (letters and numbers).
 * It may contain alphanumeric characters, underscores (_) and dashes (-).
 * Each underscore or dash must be preceded by an alphanumeric character.
 * The name must be at least two characters long.
 
 Each pipeline must have its own target table in the data lake. Thus, you must select distinct target table names for each offloading configuration.
 
-For the **alarms**, **events**, and **inventory** collections, you only need to specify the target table name in this step.
+For the **alarms**, **events**, and **inventory** collections, you must only specify the target table name in this step.
```
Comment (BeateRixen) [inline content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md:60]: see above

File: content/microservice-sdk/java-bundle/client-library.md [hunk 5]
```diff
@@ -202,7 +202,7 @@ operation.set(relay);
 control.create(operation);
 ```
 
-Now, if you would like to query the pending operations from an agent, the following code would need to be executed:
+Now, if you would like to query the pending operations from an agent, the following code must be executed:
```
Comment (BeateRixen) [inline content/microservice-sdk/java-bundle/client-library.md:205]: would like to > perhaps better "want to"

File: content/smartrest/handling-of-ids.md [hunk 6]
```diff
@@ -31,13 +31,13 @@ This removes the necessity to tell the ID to the device, and because the client
...
-When a client creates an alarm using the REST API, it needs to ensure that it gets the ID of the alarm that was generated by {{< product-c8y-iot >}} in return.
+When a client creates an alarm using the REST API, it must ensure that it gets the ID of the alarm that was generated by {{< product-c8y-iot >}} in return.
 
 The client will need this ID to later update the alarm, for example, to status CLEARED, if the alarm is not active anymore.
 
 ![REST Handling Alarms](/images/mqtt/exampleRestAlarmId.png)
 
 In {{< product-c8y-iot >}}, a device can only have a single alarm per type in status ACTIVE. If it creates another alarm with the same type, it will get de-duplicated.
-Therefore, the MQTT implementation uses the type of an alarm as identifier. The client only needs to send which type of alarm has been resolved, and the server will find the correct alarm object.
+Therefore, the MQTT implementation uses the type of an alarm as identifier. The client must only send which type of alarm has been resolved, and the server will find the correct alarm object.
```
Comment (BeateRixen) [inline content/smartrest/handling-of-ids.md:41]: see above

File: content/smartrest/mqtt-static-templates.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ weight: 60
 
 ### Overview {#overview}
 
-To ease device integration {{< product-c8y-iot >}} already supports a number of static templates that can be used by any client without the need to create your own templates. These templates focus on the most commonly used messages for device management purposes.
+To ease device integration {{< product-c8y-iot >}} already supports a number of static templates that can be used by any client without creating your own templates. These templates focus on the most commonly used messages for device management purposes.
```
Comment (BeateRixen) [inline content/smartrest/mqtt-static-templates.md:11]: I would revert this change as it slightly changes the meaning.

File: content/standard-tenant/managing-permissions-bundle/inventory-roles.md [hunk 8]
```diff
@@ -172,7 +172,7 @@ The response looks like below:
     }
     ...
 
-This allows the user to see measurements that contain only the defined types, without the additional need to configure other types of fragments that the measurement has.
```
Comment (BeateRixen) [inline content/standard-tenant/managing-permissions-bundle/inventory-roles.md:175]: See above, please revert

**Review by BeateRixen — APPROVED** [review]

**Review by kailol-sag — APPROVED** [review]
Approving the changes for Streaming Analytics.

**Review by MWindel — APPROVED** [review]
